docs/sources/developers/plugins/publishing-and-signing-criteria.md

```diff
@@ -36,8 +47,6 @@ Plugins must be licensed under one of the following AGPL compliant licenses for
 
 If contributing a plugin on behalf of an organization, be sure to seek guidance from your legal team.
 
-## Commercial
-
-- Usage of 3rd party software or dependencies within the plugin must be licensed for the intended use. For example, using open source dependencies must be credited/licensed; embedding logos or trademarks;
+## Commercial licensing
 
-Grafana Labs reserves the right to decline or remove any plugin at its discretion. Failure to comply with publishing and signing criteria may result in immediate removal from the Grafana plugin catalog.
+Usage of third party software or dependencies within the plugin must be licensed for the intended use. For example, using open source dependencies must be credited/licensed; and embedding logos or trademarks.
```

Comment:
Not sure what the bit after the semi-colon means

Reply:
Done in PR 67070.